src/shared/components/search-result/search-result.component.ts

```diff
@@ -48,6 +48,9 @@ export class SearchResultComponent implements OnChanges {
   @Output()
   changePage: EventEmitter<any> = new EventEmitter();
 
+  @Output()
+  clickCase: EventEmitter<any> = new EventEmitter();
```

Comment:
It is a new Emitter. Who is listening to that? Or is it something planned for the future?